### DISCUSSION TITLE
Drastic

### DISCUSSION THREAD

**Discussion by Andrewonthetop** [OnionUI/Onion#1444] [Support]
Hello everyone 
I have a problem on my miyoo once I launch it it remains block on a screen this makes me sa since the last onion update 
would you have an idea
![IMG_0266](https://github.com/OnionUI/Onion/assets/161372000/c798091d-dcad-4a21-bc7a-8ed20d16ac51)

**Comment by Aemiii91** [comment]
Try deleting the affected save state in `Saves/CurrentProfile/states/Drastic`.

**Reply by Andrewonthetop** [reply]
yes the problem is that every time I delete a file on the SD card the one if comes back
